src/plugin.ts

```diff
@@ -0,0 +1,103 @@
+import { FastifyLoggerInstance, FastifyPluginAsync } from 'fastify';
+import { Actor, Item, Member, PermissionLevel, UnknownExtra } from 'graasp';
+import { ItemTagTaskManager, ItemTagService } from 'graasp-item-tags';
+import { isGraaspError } from './utils';
+
+export interface GraaspHiddenOptions {
+  hiddenTagId: string;
+}
+
+const plugin: FastifyPluginAsync<GraaspHiddenOptions> = async (fastify, options) => {
+  const {
+    items: { dbService: iS, taskManager: itemTaskManager },
+    itemMemberships: { dbService: iMS, taskManager: membershipTaskManager },
+    taskRunner: runner,
+  } = fastify;
+
+  const { hiddenTagId } = options;
+
+  const iTS = new ItemTagService();
+  const taskManager = new ItemTagTaskManager(iS, iMS, iTS, itemTaskManager);
+
```

Comment:
Add a comment that it is hidden only if the user does not have admin permission over it.